configuration/file.go

```diff
@@ -38,6 +40,16 @@ func loadFile(path string) (yamlConf []byte, err error) {
 	return yamlConf, nil
 }
 
+// writeFile writes the configuration yaml into the
+// specified path
+func writeFile(filename string, content []byte) error {
```

Comment:
Is this function really necessary.  We have 10 lines of code that allow you to omit one parameter.